documents/LORE.md

```diff
@@ -0,0 +1,23 @@
+# The Stickian Empire
+
+## Topics
+
+1. Many years ago, the rubberian people were defeated at the Great War of the Rubberhand
+2. All realms across the now called Rubberfree land were living in peace, focused on prosperity and growth
+3. The king Madstick, the III, was not happy. Despite being decorated as the strongest general, he didn't receive a great price when the war ended.
+4. He waited for some years until the union between realms was not as strong as before, during the war. 
```

Comment:
how many years tho

Reply:
I guess that for now that we can just define time relatively to events. However, later on this might be important to define specific dates (to avoid inconsistencies).